docs/source/causal-lm.rst

```diff
@@ -7,10 +7,12 @@ Base Classes
 .. autoclass:: curated_transformers.models.CausalLMModule
    :members:
    :show-inheritance:
+   :inherited-members: Module
```

Comment:
Doesn't this add a lot of methods from PyTorch? Maybe we want `TransformerModule` here?

Reply:
My understanding of the [RST docs](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) is that the classes that follow the directive are _excluded_ from the member list? At least it works as intended locally (from my testing).